dts/nxp/mcx/MCXW716CMFPA-pinctrl.h

```diff
@@ -0,0 +1,181 @@
+/*
+ * Copyright 2024 NXP
```

Comment:
Missing the SPDX License header. Did you generate this file, or add the copyright manually? I think the generation script will make this file Apache-2.0 licensed, so let's use that

Reply:
updated